rpm/pacemaker.spec.in

```diff
@@ -183,18 +183,6 @@
 %endif
 
 
-# Definitions for backward compatibility with older RPM versions
-
-## Ensure the license macro behaves consistently (older RPM will otherwise
-## overwrite it once it encounters "License:"). Courtesy Jason Tibbitts:
-## https://pkgs.fedoraproject.org/cgit/rpms/epel-rpm-macros.git/tree/macros.zzz-epel?h=el6&id=e1adcb77
-%if !%{defined _licensedir}
-%define description %{lua:
-    rpm.define("license %doc")
-    print("%description")
-}
-%endif
-
```

Comment:
It might be worth documenting the oldest supported version of rpm-build, say in INSTALL.md.

Reply:
Good idea. I was too lazy to look it up earlier :) %license was added in 4.11 in January 2013 (in RHEL 7, Fedora 20ish).